src/shared/components/Header/SearchBar/component.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Container, Row, Col } from 'react-bootstrap';
+import { SearchBarWrapper, LabelIcon, FormWrapper, ButtonClose } from './styles';
+import SearchIcon from '../../../icons/Search';
+import CloseIcon from '../../../icons/Close';
```

Comment:
let's create `index.ts` in icons folder - add export all icons from there

```ts
import SearchIcon from './Search';
import CloseIcon from './CloseIcon';

export {SearchIcon, CloseIcon};

```

and then here
```tsx
import { SearchIcon,  CloseIcon} from '../../../icons';
```